src/ui/load.js

```diff
@@ -33,16 +33,19 @@ export async function load(): Promise<LoadResult> {
   ];
   const responses = await Promise.all(queries);
 
-  for (const response of responses.slice(0, 4)) {
+  for (const response of responses.slice(1, 4)) {
     if (!response.ok) {
       console.error(response);
       return {type: "FAILURE", error: response.status};
     }
   }
   try {
-    const json = await responses[0].json();
-    const credResult = credResultFromJSON(json);
-    const credView = new CredView(credResult);
+    let credView = null;
+    if (responses[0].ok) {
+      const json = await responses[0].json();
+      const credResult = credResultFromJSON(json);
+      credView = new CredView(credResult);
+    }
```

Comment:
Definitely out of scope for this PR, but I'm looking forward to refactoring response parsing sequence into something more readable.